content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -222,50 +164,123 @@ The drawbacks of ORMs are not always apparent when you start using them. This se
+| Data Mapper     | The code which transforms between the object schema and the database | Mapper classes                                 | Generated functions in Prisma Client | @map attributes in the Prisma schema |
+| Database schema | The structure of data in the database, e.g., tables and columns      | SQL written by hand or with a programmatic API | SQL generated by Prisma Migrate      | Prisma schema                        |
 
-Both Prisma Client and Prisma Migrate rely on the [Prisma schema](../../components/prisma-schema). The Prisma schema is a declarative representation of your database schema. It serves as the single source truth for both the database schema and your application models.
+Prisma aligns with the Data Mapper pattern with the following added benefits:
 
-### How Prisma compares
+- Reducing the boilerplate of defining classes and mapping logic by generating Prisma Client based on the Prisma schema.
+- Eliminating the synchronization challenges between application objects and the database schema.
+- Database migrations are a first-class citizen as they're derived from the Prisma schema.
 
-Unlike ORMs, with Prisma you don't create model classes nor do you map fields in your codebase to database fields. Instead, you use the generated Prisma Client API like an advanced query builder that returns plain JavaScript objects. Schema changes are synced automatically from the models in the Prisma schema to the database.
+Now that we've talked about the concepts behind Prisma's approach to Data Mapper, we can go through how the Prisma schema works in practice.
 
-Models in Prisma mean something slightly different to ORMs. When using ORMs, models are represented as _classes_. With Prisma, models are defined in the Prisma schema as abstract entities which describe the relationships between tables:
+### Prisma schema
+
+At the heart of Prisma's implementation of the Data Mapper pattern is the _Prisma schema_ – a single source of truth for the following responsibilities:
+
+- Configuring how Prisma connects your database.
+- Generating Prisma Client – the type-safe ORM for use in your application code.
+- Creating and evolving the database schema with Prisma Migrate.
+- Defining the mapping between application objects and database columns.
+
+Models in Prisma mean something slightly different to Active Record ORMs. With Prisma, models are defined in the Prisma schema as abstract entities which describe tables, relations, and the mappings between columns to properties in the Prisma Client.
+
+As an example, here's a Prisma schema for a blog:
 
 ```prisma
-model User {
-  id        Int      @default(autoincrement()) @id
-  birthDate DateTime
-  email     String   @unique
-  firstName String?
-  lastName  String?
-  posts     Post[]
+datasource db {
+  provider = "postgres"
```

Comment:
Default is still `postgresql`: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference/#datasource